docsrc/hello_world.rst

```diff
@@ -183,9 +179,34 @@ Accessor functions extract these as:
         print(f'{k}\t{v.shape}')
 
 
+In addition to the MCMC sample itself, the CmdStanMCMC object provides
+access to the the per-chain HMC tuning parameters from the NUTS-HMC adaptive sampler,
+(if present).
+
+.. ipython:: python
+
+    fit.metric_type
+    fit.metric
+    fit.step_size
+
+
+
+The CmdStanMCMC object also provides access to metadata about the model and the sampler run.
+
+.. ipython:: python
+
+    fit.metadata.cmdstan_config.keys()
+    fit.metadata.cmdstan_config['num_samples']
+    fit.metadata.cmdstan_config['seed']
+
+    fit.metadata.stan_vars_cols.keys()
+    fit.metadata.method_vars_cols.keys()
```

Comment:
I think if you want to display these results in the build docs you need to wrap each in a `print()`, otherwise it will just show the last one as the output. Same for the block just above this

Reply:
are you sure?
<img width="1384" alt="Untitled" src="https://user-images.githubusercontent.com/5466612/175751502-19a93561-f7e7-4b2f-bbb7-b384a8b96f9e.png">